dissect/cstruct/types/structure.py

```diff
@@ -403,6 +402,18 @@ def __repr__(self) -> str:
 
         return f"<{self.__class__.__name__} {' '.join(values)}>"
 
+    @cached_property
+    def sizes(self) -> None:
```

Comment:
We can't really add a `sizes` property since it will clash with structures that have a field names `sizes`. Which is why almost everything related to these classes is either `_` or `__` 😄.

Reply:
Ah yes, a similar mistake as the  `default` method IIRC. 

I renamed to `__field_sizes__`